x-pack/docs/en/rest-api/security/get-service-accounts.asciidoc

```diff
@@ -65,7 +65,8 @@ GET /_security/service/elastic/fleet-server
     "role_descriptor": {
       "cluster": [
         "monitor",
-        "manage_own_api_key"
+        "manage_own_api_key",
+        "read_fleet_secrets"
```

Comment:
as a follow up, do we want to provide additional documentation?

for example:
- document the privileges in https://www.elastic.co/guide/en/elasticsearch/reference/master/security-privileges.html
- add docs pages for the new APIs in  https://www.elastic.co/guide/en/elasticsearch/reference/master/fleet-apis.html

Reply:
Do you know where to raise a request to update ES docs?

Reply:
@tvernum mentioned that these APIs are internal and will only be used by Kibana and the fleet server.
in that case there might be no urgency in documenting them, so I think it's fine to defer this @juliaElastic .